packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -423,21 +424,16 @@ export const signTransaction =
             enhancedTxInfo.useDecreaseOutput = hasDecreasedOutput;
         }
 
-        if (
-            account.networkType === 'ethereum' &&
-            !isCardanoTx(account, enhancedTxInfo) &&
-            enhancedTxInfo.token?.contract &&
-            network?.chainId
-        ) {
-            const isTokenKnown = await fetch(
-                `https://data.trezor.io/firmware/eth-definitions/chain-id/${
-                    network.chainId
-                }/token-${enhancedTxInfo.token.contract.substring(2).toLowerCase()}.dat`,
-            )
-                .then(response => response.ok)
-                .catch(() => false);
-
-            enhancedTxInfo.isTokenKnown = isTokenKnown;
+        if (account.networkType === 'ethereum' && !isCardanoTx(account, enhancedTxInfo)) {
```

Comment:
I think that the correct condition should be:

`(account.networkType === 'ethereum' && enhancedTxInfo.token?.contract)`

Reply:
Oh I will check, that's probably some wrong conflict resolution

Reply:
52ef3043a0d6e73f882cbc48a2a12690e521897f

Reply:
I am almost sure that checking !isCardanoTx(account, enhancedTxInfo) is not needed since it just returns if account.networkType === 'cardano' and we have a condition to check account.networkType === 'ethereum'.

Reply:
This cardano thing is all over send form actions